docs/operating-eck/licensing.asciidoc

```diff
@@ -9,6 +9,8 @@ endif::[]
 
 When you install the default distribution of ECK, you receive a Basic license. Any Elastic stack application you manage through ECK will also be Basic licensed. Go to https://www.elastic.co/subscriptions to see which features are included in the Basic license for free.
 
+IMPORTANT: ECK is only offered in two licensing tiers: Basic and Enterprise. Similar to the Elastic Stack, customers are able to download and use ECK with a Basic license for free. Basic license users can avail of support from GitHub or through our link:https://discuss.elastic.co[community]. A paid Enterprise subscription is required to engage the Elastic support team. For more detail on Enterprise subscriptions, see link:https://www.elastic.co/subscriptions[here].
```

Comment:
```suggestion
IMPORTANT: ECK is only offered in two licensing tiers: Basic and Enterprise. Similar to the Elastic Stack, customers are able to download and use ECK with a Basic license for free. Basic license users can avail of support from GitHub or through our link:https://discuss.elastic.co[community]. A paid Enterprise subscription is required to engage the Elastic support team. For more details, see the link:https://www.elastic.co/subscriptions[Elastic Stack subscriptions].
```

Reply:
HI @alaudazzi 
I'm not sure about using the title - Elastic Stack subscriptions for the subscription link. 
In my head, the stack subscriptions would be a link to the Gold / Platinum i.e. Standalone Elastic On-Prem, when we want them to be thinking of Orchestration Elastic On-Prem as shown in the screenshot. 

<img width="766" alt="subscriptions" src="https://user-images.githubusercontent.com/36622494/78597921-3cb8a880-7846-11ea-83fa-e2f6fe148ea5.png">

What do you think?